library_conventions/packages.adoc

```diff
@@ -193,6 +193,66 @@ image:img/footprint_pin1_sot669.png[]
 ====
 
 
+[#libraryconventions-packages-placement]
+==== Placement Layer
+
+The _Top Placement_ layer is intended to be printed on silkscreen and thus
+should contain information required for assembling the PCB. But don't put too
+many things on that layer as it would waste space on the PCB!
+
+Typically this layer should only contain some lines and dots to indicate where
+and in which orientation the device gets assembled, for example an outline and
+a dot next to pin 1.
+
+*The placement should be drawn according {ipc7351c-slides}[IPC-7351C]*. The
```

Comment:
> IPC-7351C

That standard is not out yet, maybe that deserves a note.

Reply:
Does that matter? IPC-7351C is mentioned multiple times, IMHO it's not reasonable to mention that every time. At least it's a direct hyperlink to the PDF, so one could even write "according this document" :grin:

Reply:
Ok let's stick it :slightly_smiling_face: